tests/integration/targets/ndo_macsec_policy/tasks/main.yml

```
@@ -0,0 +1,402 @@
+        security_policy: 'should_secure'
+        sak_expiry_time: 99
+        confidentiality_offset: 50
+        key_server_priority: 11
+        interface_type: access
+        macsec_keys:
+          - key_name: abc12
+            psk: 'AA111111111111111111111111111111111111111111111111111111111111aa'
+            start_time: '2029-12-11 11:12:13'
+            end_time: '2030-12-11 11:12:13'
+          - key_name: ABC
+            psk: 'AAabcdabcdabcdabcdabcdabcdabcdab11111111111111111111111111111aaa'
+        state: present
+      register: nm_add_macsec_policy_2
+
+    - name: Update the MACsec policy of interface_type 'access' (check mode)
+      cisco.mso.ndo_macsec_policy: &update_macsec_policy
+        <<: *add_macsec_policy_2
+        description: 'Updated description'
+        admin_state: 'disabled'
+        cipher_suite: '128_gcm_aes'
+        window_size: 110
+        security_policy: 'must_secure'
+        sak_expiry_time: 100
+        confidentiality_offset: 30
+        key_server_priority: 10
+        macsec_keys:
+          - key_name: abc12
+            psk: 'AAabcdabcdabcdabcdabcdabcdabcdab'
+            start_time: '2029-12-11 11:12:13'
+          - key_name: ABC
+            psk: 'AAabcdabcdabcdabcdabcdabcdabcdab'
+          - key_name: aaa11
+            psk: 'AAabcdabcdabcdabcdabcdabcdabcdab'
+            start_time: '2025-10-10 10:12:13'
+            end_time: '2026-10-10 10:12:13'
+        state: present
+      check_mode: true
+      register: cm_update_macsec_policy
+
+    - name: Update the MACsec policy of interface_type 'access'
+      cisco.mso.ndo_macsec_policy:
+        <<: *update_macsec_policy
+      register: nm_update_macsec_policy
+
+    # Idempotency for update cannot be checked as the psk is encrypted and the encrypted value changes every time
+    - name: Update MACsec policy of interface_type 'access' again
+      cisco.mso.ndo_macsec_policy:
+        <<: *update_macsec_policy
+      ignore_errors: true
```

why is this ignore errors here?